### PR TITLE
feat: mf-4330 new wallet assets in popup

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-4330
https://mask.atlassian.net/browse/MF-4466
https://mask.atlassian.net/browse/MF-4468

### CODE REVIEW

File: packages/mask/src/extension/popups/components/PopupLayout/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                },
+            },
+        }}
+    />
+)
+
+const useStyles = makeStyles()({
+    container: {
+        height: '100%',
+        minHeight: 560,
```
Comment (nuanyang233) [inline packages/mask/src/extension/popups/components/PopupLayout/index.tsx]: The `minHeight` for the popup in the new design draft is 620.

File: packages/mask/src/extension/popups/components/PopupLayout/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+export const PopupLayout = memo(function PopupLayout({ children }: PropsWithChildren<{}>) {
+    const { classes } = useStyles()
+
+    const location = useLocation()
+    const matched = PATTERNS.some((pattern) => matchPath(pattern, location.pathname))
+
+    return (
+        <>
+            {GlobalCss}
+            <Paper elevation={0} style={{ height: '100vh', overflowY: 'auto', minHeight: 560, borderRadius: 0 }}>
```
Comment (nuanyang233) [inline packages/mask/src/extension/popups/components/PopupLayout/index.tsx]: same as above.

File: packages/mask/src/extension/popups/pages/Wallet/components/AssetsList/index.tsx [hunk 3]
```diff
@@ -1,40 +1,54 @@
...
+        fontSize: 16,
+        fontFamily: 'Helvetica',
+        fontWeight: 700,
+        color: theme.palette.maskColor.main,
     },
-    balance: {
-        fontSize: 14,
+    value: {
+        fontSize: 16,
+        fontFamily: 'Helvetica',
```
Comment (nuanyang233) [inline packages/mask/src/extension/popups/pages/Wallet/components/AssetsList/index.tsx:50]: Maybe we need to set the font family to global.

File: packages/mask/src/extension/popups/components/PopupLayout/index.tsx (hunk 1 above)
Comment (UncleBill) [reply packages/mask/src/extension/popups/components/PopupLayout/index.tsx]: OK, would set to 600 which is max height of popup limited by Chrome.
https://developer.chrome.com/docs/extensions/reference/browserAction/#popup

File: packages/mask/src/extension/popups/pages/Wallet/components/AssetsList/index.tsx (hunk 3 above)
Comment (UncleBill) [reply packages/mask/src/extension/popups/pages/Wallet/components/AssetsList/index.tsx:50]: Yep.

**Review by guanbinrui — APPROVED** [review]

**Review by nuanyang233 — APPROVED** [review]

**Pull request merged**